src/plugins/kibana/public/dashboard/services/_saved_dashboard.js

```diff
@@ -56,6 +57,7 @@ module.factory('SavedDashboard', function (courier, config) {
     timeRestore: 'boolean',
     timeTo: 'string',
     timeFrom: 'string',
+    refreshInterval: 'string'
```

Comment:
So i would like to just save the millis for the refresh, but i really think we should save the whole thing. The reason being is there is other information stored there, such as whether or not the timer is paused which i can definitely see someone wanting to use. Also even the Words that represent the refresh are customizable.

I think in order to save just the millis it would require a change in structure to the refresh interval.

I think perhaps just stringifying some json would do what i need here.